SimG4Core/CustomPhysics/src/G4muDarkBremsstrahlungModel.cc

```diff
@@ -265,11 +263,7 @@ G4double G4muDarkBremsstrahlungModel::ComputeCrossSectionPerAtom(
   G.function = &DsigmaDx;
   G.params = &alpha;
   G4double xmin = 0;
-  G4double xmax = 1;
-  if ((muonMass / E0) > (MA / E0))
-    xmax = 1 - muonMass / E0;
-  else
-    xmax = 1 - MA / E0;
+  G4double xmax = muonMass > MA ? muonMass / E0 : MA / E0;
```

Comment:
This changes the value of `xmax`, doesn't it? Before it was
```c++
  if ((muonMass / E0) > (MA / E0))
    xmax = 1 - muonMass / E0;
  else
    xmax = 1 - MA / E0;
```
and now it's equivalent to
```c++
  if ((muonMass / E0) > (MA / E0))
    xmax = muonMass / E0;
  else
    xmax = MA / E0;
```